release-notes/3.0/preview/api-diff/preview5/standalone-packages/3.0-preview5-standalone-packages.md

```diff
@@ -1,4 +1,4 @@
-# API Difference netcoreapp3.0-preview4 vs netcoreapp3.0-preview5
+# API Difference standalone-preview4 vs standalone-preview5.
```

Comment:
nit: remove the `.` from the end of the title. It doesn't exist elsewhere and hence is inconsistent.

Same with the preview4-standalone-packages.md above.